### PR TITLE
Simplify documentation

### PR DESCRIPTION
**Pull request by bpradipt**
Signed-off-by: Pradipta Banerjee <pradipta.banerjee@gmail.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/139#" title="Author self-approved">bpradipt</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: README.md
Comment (yussufsh) [inline README.md]: We do not have windows support for PowerVC yet. It is still pending in the issues list.

File: docs/quickstart.md
Comment (yussufsh) [inline docs/quickstart.md]: parallelism is not needed

File: docs/var.tfvars-doc.md
Comment (yussufsh) [inline docs/var.tfvars-doc.md:25]: Default -> <domain_name>

File: var.tfvars
Comment (yussufsh) [inline var.tfvars]: Value "<subscription-id>" wont work for centos.. should be "" or code needs to be updated.

**Review by yussufsh — COMMENTED** [review]
First batch of review comments.

File: README.md
Comment (bpradipt) [reply README.md]: Ok.. I'll remove it

File: docs/quickstart.md
Comment (bpradipt) [reply docs/quickstart.md]: Yep..Thanks for catching it..

File: docs/var.tfvars-doc.md
Comment (bpradipt) [reply docs/var.tfvars-doc.md:25]: Does PowerVC support any different domain name today ?

File: var.tfvars
Comment (bpradipt) [reply var.tfvars]: Ideally the code should also handle empty string. However currently this should work since it's not empty string. Do you see any issue with this ?

**Comment by bpradipt** [comment]
@yussufsh @sudeeshjohn addressed all your comments. ptal

File: var.tfvars
Comment (yussufsh) [reply var.tfvars]: My concern is if user is using CentOS  and leave `rhel_subscription_username` as is then  https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/modules/1_bastion/bastion.tf#L162 will go ahead and register the centos machine which we don't want.

File: docs/var.tfvars-doc.md
Comment (yussufsh) [reply docs/var.tfvars-doc.md:25]: I am not sure... @sudeeshjohn ?

Comment (bpradipt) [reply docs/var.tfvars-doc.md:25]: From this doc "Default" is the only supported value - https://www.ibm.com/support/knowledgecenter/SSVSPA_1.4.4/com.ibm.powervc.cloud.help.doc/powervc_pg_authenticatingwithpowervc_cloud.html

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/139#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/139#pullrequestreview-533571019" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by sudeeshjohn — APPROVED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/139#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/139#pullrequestreview-533579966" title="Approved">sudeeshjohn</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervm/pull/139#pullrequestreview-533571019" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervm).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervm/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/ocp_prereqs_powervc.md
Comment (pravin-dsilva) [inline docs/ocp_prereqs_powervc.md:32]: Should this line be for _bastion_ instead of _worker_ ?

Comment (bpradipt) [reply docs/ocp_prereqs_powervc.md:32]: @pravin-dsilva there could be reasons to have more cpu/mem for bastion for improved performance of haproxy and NFS.

Comment (yussufsh) [reply docs/ocp_prereqs_powervc.md:32]: @bpradipt there is a typo in the statement for bastion mentioning as "worker".

`Increase worker vCPUs` -> `Increase bastion vCPUs`

**Pull request merged**
